### ISSUE TITLE
Add Base for onchain Passports

### ISSUE THREAD

**Issue by Jkd-eth** [passportxyz/passport#3118]
### User Story:
As a Passport holder,
I want to push my onchain Passport to the Base chain,
So that I can utilize my identity and credentials seamlessly across multiple blockchains and enhance my interoperability within the crypto ecosystem.

### Acceptance Criteria
GIVEN I am a user with a Passport,
WHEN I choose to push my Passport to Base via the Passport interface,
THEN my Passport should be successfully pushed to the Base network and verifiable through Base compatible services.

[EAS Base](https://github.com/ethereum-attestation-service/eas-contracts?tab=readme-ov-file#deployments) details 

#### Tech Details:

- Integration of Base as a supported network in the existing Passport infrastructure.
- Base - 0xc5093228529547272bE5E58db15a5d36fb2C0424 - multisig in SAFE for Passport

**Comment by nutrina** [comment]
@Jkd-eth  we'll need a safe for this

**Comment by Jkd-eth** [comment]
@nutrina Yep, it's in there under tech details :) Already validated so we're good to go when we're ready

**Comment by lebraat** [comment]
Docs are ready and published (I know this is premature, but it's low risk that anyone is going to stumble upon this before we launch)
